prow/cmd/jenkins-operator/main.go

```diff
@@ -100,6 +100,11 @@ func main() {
 		ghc = github.NewClient(oauthSecret, *githubEndpoint)
 	}
 
+	logger := logrus.StandardLogger()
+	kc.Logger = logger.WithField("client", "kube")
+	jc.Logger = logger.WithField("client", "jenkins")
+	ghc.Logger = logger.WithField("client", "github")
```

Comment:
It may be worth putting `gc.logger = logrus.StandardLogger().WithField("client", "github")` into `github.NewClient`. The `Logger` field doesn't really need to be public, and this helps make sure that things are consistent going forward.

I'd be okay with that happening in a separate PR.

Reply:
Yeah, separate PR.